src/DependencyInjection/Factory/MenuAdminFactory.php

```diff
@@ -38,6 +38,7 @@ public function addConfiguration(NodeBuilder $builder)
     {
         $builder
             ->booleanNode('recursive_breadcrumbs')->defaultTrue()->end()
+            ->booleanNode('use_burgov_keyvalue_form')->defaultFalse()->end()
```

Comment:
plesae make this a true/false/auto option node like in the [seo configuration](https://github.com/symfony-cmf/seo-bundle/blob/5900c572b10f9df58689e898feb1970bd381ef43/src/DependencyInjection/Configuration.php#L298-L306)